lib/pry/prompt.rb

```diff
@@ -140,6 +141,8 @@ def prompt_name(name)
       )
     end
 
-    add(:none, 'Wave goodbye to the Pry prompt.', Array.new(2)) { '' }
+    add('none',
```

Comment:
Is highlighting also broken in this example?

Reply:
nope, this works fine, i changed it to be consistent with the other cases using add.